dockerfiles/nginx-tls/docker-entrypoint.sh

```diff
@@ -14,13 +14,17 @@ log_format json_event '{ \"@timestamp\": \"\$time_iso8601\", '
                          '\"body_bytes_sent\": \$body_bytes_sent, '
                          '\"bytes_sent\": \$bytes_sent, '
                          '\"request_time\": \$request_time, '
-                         '\"upstream_response_time\": \"\$upstream_response_time\", '
                          '\"upstream_addr\": \"\$upstream_addr\", '
                          '\"upstream_status\": \"\$upstream_status\", '
+                         '\"upstream_response_time\": \"\$upstream_response_time\", '
                          '\"upstream_response_length\": \"\$upstream_response_length\", '
                          '\"upstream_bytes_received\": \"\$upstream_bytes_received\", '
+                         '\"upstream_bytes_sent\": \"\$upstream_bytes_sent\", '
+                         '\"upstream_connect_time\": \"\$upstream_connect_time\", '
+                         '\"upstream_queue_time\": \"\$upstream_queue_time\", '
+                         '\"upstream_session_time\": \"\$upstream_session_time\", '
+                         '\"upstream_header_time\": \"\$upstream_header_time\", '
                          '\"upstream_cache_status\": \"\$upstream_cache_status\", '
-                         '\"upstream_cookie_name\": \"\$upstream_cookie_name\", '
```

Comment:
Did you intentionally remove this? There's nothing in your PR description or commit message which mentions removing fields so I just wanted to check if tis was a mistake?

Reply:
Yes, I added upstream_cookie_name but realised that it had no value because according to Nginx documentation upstream_cookie_`name` needs to be defined e.g.  upstream_cookie_`x_govuk_session_cookie`.